File: backend/src/main/java/com/se114p12/backend/dto/response/response/NotificationResponse.java
package com.se114p12.backend.dto.response.response;

import com.se114p12.backend.enums.NotificationType;
import lombok.Data;
import org.springframework.lang.Nullable;

import java.time.Instant;

@Data
public class NotificationResponse {
    private Long id;
    private NotificationType type;
    private String title;
    private String message;

    @Nullable
//    private String anchor;
    private Integer status;
    private Instant createdAt;
}

File: frontend/MAM/app/src/main/java/com/example/mam/dto/notification/NotificationResponse.kt
package com.example.mam.dto.notification

import com.example.mam.dto.BaseResponse
import com.example.mam.entity.Notification
import java.time.Instant

data class NotificationResponse(
    val type: String = "",
    val title: String = "",
    val message: String = "",
    val status: Int = 0,
    val userId: Long = 0L,
    val isRead: Boolean = false
): BaseResponse()
